api/handlers/github.js

```diff
@@ -58,12 +58,24 @@ const github = module.exports = (() => {
         const octokit = new Octokit({
             auth: params.auth_key
         })
-        const res = await octokit.repos.listForAuthenticatedUser({
-            affiliation: ['owner'],
-            per_page: 100,
-            sort: 'updated',
-            direction: 'desc',
-        })
+        let res
```

Comment:
@mecoccaro We want to use `const` whenever it's possible, so refactor this code to maybe something like `const res = {}` and then assign the values that you consider that are relevant to the object structure. I prefer having multiple `const` variables than just a few using `let` or `var`

@otech47 shared with me once an article about why we should enforce `const` over `let` and `var` but I couldn't find it, maybe he still has it pinned somewhere

Reply:
https://codeburst.io/javascript-var-let-or-const-which-one-should-you-use-2fd521b050fa

Reply:
The thing here is that as the article says I know that the value will change, if I change it to const in this case, when we change the org it will cause an error because of the change of value on the constant.